embrace-android-delivery/src/main/kotlin/io/embrace/android/embracesdk/internal/delivery/SupportedEnvelopeType.kt

```diff
@@ -8,13 +9,14 @@ import java.lang.reflect.Type
  */
 enum class SupportedEnvelopeType(
```

Comment:
Need to know which endpoint an envelope type can go to to implement the logic to block sending to specific endpoints if the server returns a 429.